include/clang/StaticAnalyzer/Core/PathDiagnosticConsumers.h

```diff
@@ -21,17 +21,19 @@ namespace clang {
 
 class AnalyzerOptions;
 class Preprocessor;
+namespace cross_tu {
+class CrossTranslationUnitContext;
+}
 
 namespace ento {
 
 class PathDiagnosticConsumer;
 typedef std::vector<PathDiagnosticConsumer*> PathDiagnosticConsumers;
 
-#define ANALYSIS_DIAGNOSTICS(NAME, CMDFLAG, DESC, CREATEFN)\
-void CREATEFN(AnalyzerOptions &AnalyzerOpts,\
-              PathDiagnosticConsumers &C,\
-              const std::string &Prefix,\
-              const Preprocessor &PP);
+#define ANALYSIS_DIAGNOSTICS(NAME, CMDFLAG, DESC, CREATEFN)                    \
```

Comment:
Please avoid unrelated changes.

Reply:
The whole macro ANALYSIS_DIAGNOSTICS was changed, and clang-format updated the formatting for it. (Other option is to add new things with the existing "bad" formatting.)

Reply:
As I see, we just added a new parameter, but previously I missed that. So, OK.